.gitignore

```diff
@@ -0,0 +1,2 @@
+go.work
+go.work.sum
```

Comment:
Do we still need this, considering https://github.com/celestiaorg/go-square/pull/67 was merged?

Reply:
I think so, else it's going to see this as a diff between the users branch and main and will want to check it in